README.md

```diff
@@ -32,16 +32,16 @@ $ gem install ledger_sync
 
 To use LedgerSync, you must create an `Operation`.  The operation will be ledger-specific and will require the following:
 
-1. Adaptor
+1. Ledger
 2. Resource(s)
 
 The code may look like the following:
 
 ```ruby
-# First we create an adaptor, which is our connection to a ledger.
+# First we create an connection, which is our connection to a ledger.
 # Each ledger may require different keys, so check the
 # documentation below for specifics.
-adaptor = LedgerSync::Adaptors::QuickBooksOnline::Adaptor.new(
+connection = LedgerSync::Ledgers::QuickBooksOnline::Ledger.new(
```

Comment:
I'm not sure I'm fan of this naming convention. Why not to name it `LedgerSync::Ledgers::QuickBooksOnline::Connection.new(`